src/resources/js/frontend-ticket-form.js

```diff
@@ -0,0 +1,258 @@
+/**
+ * Provides global stock handling for frontend ticket forms.
+ *
+ * @var object tribe_tickets_stock_data
+ */
+jQuery( function( $ ) {
```

Comment:
Since this is a new file, should we take the opportunity to expose the functions and vars globally (adding them as properties of a globally accessible object)? That way we can do a bit more testing in the console and allow plugin authors to override/modify the functionality provided by this file.

The [RSVP js file](https://github.com/moderntribe/event-tickets/blob/release/122/src/resources/js/rsvp.js) is an example of what I mean.